.travis.yml

```diff
@@ -1,14 +1,19 @@
 language: csharp
 
+dist: trusty
```

Comment:
Have never needed that... it does say "All Trusty images have sudo enabled" so maybe that explains why it's working? I'm happy to modify it if you'd like, but it does seem to be okay...